logic/src/commonMain/kotlin/com/wire/kalium/logic/data/auth/login/SSOLoginRepository.kt

```diff
@@ -3,9 +3,9 @@ package com.wire.kalium.logic.data.auth.login
 import com.wire.kalium.logic.NetworkFailure
 import com.wire.kalium.logic.functional.Either
 import com.wire.kalium.logic.wrapApiRequest
-import com.wire.kalium.network.api.AuthenticationResultDTO
-import com.wire.kalium.network.api.user.login.SSOLoginApi
-import com.wire.kalium.network.api.user.login.SSOSettingsResponse
+import com.wire.kalium.network.api.base.model.AuthenticationResultDTO
+import com.wire.kalium.network.api.base.unAuthenticated.SSOLogin
```

Comment:
```suggestion
import com.wire.kalium.network.api.base.unauthenticated.SSOLogin
```
Noticed that the casing wrong in a few places.